client/hotrod-client/src/test/java/org/infinispan/client/hotrod/event/ClientListenerWithFilterAndProtobufTest.java

```diff
@@ -48,24 +50,25 @@
 
    @Override
    protected void createCacheManagers() throws Throwable {
-      ConfigurationBuilder cfgBuilder = hotRodCacheConfiguration(getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC, false));
+      ConfigurationBuilder defaultClusteredCacheConfig = getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC, false);
```

Comment:
I see this PR touched ClientListenerWithFilterAndProtobufTest and I'm a bit surprised ClientListenerWithFilterAndRawProtobufTest is left unchanged. Overlooked or that the actual intention?

Reply:
No need to change that one since it uses "raw" filters/converters, but I will port over the comments